src/stm32/stm32h7_spi.c

```diff
@@ -130,14 +124,15 @@ spi_transfer(struct spi_config config, uint8_t receive_data,
     uint8_t rdata = 0;
     SPI_TypeDef *spi = config.spi;
 
-    MODIFY_REG(spi->CR2, SPI_CR2_TSIZE, len);
+    spi->CR2 = SPI_CR2_TSIZE;
```

Comment:
Seems like there should be:
```
spi->CR2 = len << SPI_CR2_TSIZE_Pos; // SPI_CR2_TSIZE_Pos (0U)
```

Reply:
Thanks for reviewing.

I must be missing something, the original code was `MODIFY_REG(spi->CR2, SPI_CR2_TSIZE, len);` and the definition of that macro is: `#define MODIFY_REG(REG, CLEARMASK, SETMASK)`.

So, the old code and new code seem the same to me.  I can add in the `<< SPI_CR2_TSIZE_Pos` (which is a noop) for documentation purposes if you think that will help.

Am I missing something?

Cheers,
-Kevin

Reply:
Oh - sorry - that was a dumb typo on my part.  (I kept "seeing" what I meant instead of what I actually wrote.)

Hopefully fixed now.

Thanks,
-Kevin